packages/okta-adapter/src/filters/user.ts

```diff
@@ -104,16 +90,20 @@ const filterCreator: FilterCreator = ({ paginator, config, usersPromise }) => {
       })
     },
     preDeploy: async (changes: Change<InstanceElement>[]) => {
-      if (config[FETCH_CONFIG].convertUsersIds === false) {
+      const { convertUsersIds, getUsersStrategy } = config[FETCH_CONFIG]
+      if (convertUsersIds === false) {
```

Comment:
This implicitly assumes that the default value of `convertUsersIds` is `true`.
We can maybe be more explicit here.
Having said that, when writing this code it feels like overkill to me, so your call :)
```suggestion
      if (convertUsersIds === false || (convertUsersIds === undefined && DEFAULT_CONVERT_USERS_IDS_VALUE === false) {
```